Library/Formula/gnupg2.rb

```diff
@@ -69,7 +67,7 @@ def install
 
     # Conflicts with a manpage from the 1.x formula, and
     # gpg-zip isn't installed by this formula anyway
-    rm man1/"gpg-zip.1"
+    rm_f man1/"gpg-zip.1"
```

Comment:
I'll leave this as is but I don't think we want to get into the habit of adding these because it means we won't remove these lines when they aren't necessary.